packages/template-retail-react-app/package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "dependencies": {
     "@chakra-ui/icons": "^2.0.19",
-    "@chakra-ui/react": "^2.6.0",
+    "@chakra-ui/react": "2.8.2",
```

Comment:
Why are we pinning the version here? As opposed to using a `^` as we did previously?

Reply:
done